src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="item" data-qa="card">
+    <img
+      src="./images/imac.jpeg"
+      alt="APPLE A1419 img"
+      class="item__image"
+    >
+    <span class="item__name">
+      Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+    </span>
+    <span class="item__code">
+      Код товара: 195434
+    </span>
+    <div class="item__rates">
+      <div class="stars">
+        <div class="stars__star star1">
+        </div>
+        <div class="stars__star star2">
+        </div>
+        <div class="stars__star star3">
+        </div>
+        <div class="stars__star star4">
+        </div>
+        <div class="stars__star star5">
+        </div>
+      </div>
+      <a href="#" class="review">
+        Отзывов: 5
+      </a>
+    </div>
+    <div class="item__price">
+      <span class="price-word">
+        Цена:
+      </span>
+      <span class="price-value">
+        69 999 грн
+      </span>
+    </div>
+    <a href="#"
+      class="item__buy-link buy-link_hover"
+      data-qa="hover"
+    >
```

Comment:
add hover styles using :hover

```suggestion
    <a
      href="#"
      class="item__buy-link"
      data-qa="hover"
    >
```